cpp/net/url_fetcher.cc

```diff
@@ -67,29 +65,30 @@ UrlFetcher::Request NormaliseRequest(UrlFetcher::Request req) {
 
 
 State::State(ConnectionPool* pool, evhttp_cmd_type verb,
-             const UrlFetcher::Request& req, UrlFetcher::Response* resp,
-             Task* task)
+             const UrlFetcher::Request& request,
+             UrlFetcher::Response* response, Task* task)
     : pool_(CHECK_NOTNULL(pool)),
-      conn_(pool_->Get(req.url)),
-      request_(NormaliseRequest(req)),
-      response_(CHECK_NOTNULL(resp)),
-      task_(CHECK_NOTNULL(task)),
-      http_req_(CHECK_NOTNULL(evhttp_request_new(&RequestCallback, this))) {
+      conn_(pool_->Get(request.url)),
+      request_(NormaliseRequest(request)),
+      response_(CHECK_NOTNULL(response)),
+      task_(CHECK_NOTNULL(task)) {
   CHECK_NOTNULL(conn_.get());
 
   if (request_.url.Protocol() != "http") {
     VLOG(1) << "unsupported protocol: " << request_.url.Protocol();
-    task->Return(Status(util::error::INVALID_ARGUMENT,
-                        "UrlFetcher: unsupported protocol: " +
-                            request_.url.Protocol()));
+    task_->Return(Status(util::error::INVALID_ARGUMENT,
+                         "UrlFetcher: unsupported protocol: " +
+                             request_.url.Protocol()));
     return;
   }
 
-  evhttp_add_header(evhttp_request_get_output_headers(http_req_), "Host",
+  evhttp_request* const req(
```

Comment:
nit: keep this as `http_req` (or `ev_req`, or ...) to provide a little more distinguishability between this and `request`?